app/controllers/find_local_council_controller.rb

```diff
@@ -50,6 +53,11 @@ def result
 
 private
 
+  def set_artefact_headers
+    artefact = fetch_artefact(BASE_PATH)
+    set_slimmer_artefact_headers(artefact)
+  end
```

Comment:
I wonder if it's more consistent for now to just hardcode them like we do with the related links.  Or if now that we're using the artefact we should also use it to populate the related links sidebar too? 

My vote would be for not using the artefact at all and hardcoding everything as a stopgap until we work out how to make this entire thing use managed content.